jenkins-pipelines/openshift/Jenkinsfile

```diff
@@ -93,6 +93,8 @@ node ('openstack-jenkins-slave') {
                 --extra-vars OPENSHIFT_USERNAME=\"\$OPENSHIFT_USERNAME\" \
                 --extra-vars OPENSHIFT_PASSWORD=\"\$OPENSHIFT_PASSWORD\" \
                 --extra-vars server=\"https://ocp.\$DOMAIN_SUFFIX:8443\"
+	    ssh -o StrictHostKeyChecking=no -i id_rsa_jenkins cloud-user@$bastionip \
+	    "cd /usr/share/ansible/openshift-deployment-ansible/tests/haproxy ; ./master-failover-test.sh \"\$ADMIN_USERNAME\" \"\$ADMIN_PASSWORD\""
```

Comment:
Do you need to change dir first? Can you not just run it with the full path since you're putting that in the command anyway?

Reply:
It doesn't have to be done in the Jenkins file it can be done in the master-failover-test script but as I need to put the full path in here anyway it seemed to make more sense to change directory at this point.

Reply:
Change directory moved out of Jenkins file and placed inside the script itself.